### ISSUE TITLE
Sharebar component should have an Omniture prop

### ISSUE THREAD

**Issue by umbobabo** [the-economist-editorial/component-world-if-app#14]
Click on an icon in the Sharebar on the MH and see the error in console.
Uncaught TypeError: Cannot read property 'linkinfo' of undefined.

The sharebar component should receive an s prop and Omniture props depending on the page.
Pls see Pagesettings on this example:
https://github.com/the-economist-editorial/component-sharebar/blob/master/example.es6

**Comment by keithamus** [comment]
This is why we need to ditch omniture as a component, and instead move to something like [react-i13n](https://github.com/yahoo/react-i13n). Our components should be generically instrumentable, and react-i13n seems like a good way to do this. Omniture should plug into us, not us into omniture.

**Comment by angelorepole** [comment]
I totally agree. However before jumping on react-i13n I need us to evaluate tealium and make sure that react-i13n is definitely the best choice for us. We are already using tealium and, unless, we have very good reason to ditch it, we won't be able (and would be just wrong) to have 2 tag managers tool

**Comment by umbobabo** [comment]
This make a lot of sense to me.
I agree that the component shouldn't have tracking functionality inside, but the tracker should monitor the component from the outside.

Anyway, at the moment, we are losing metrics required by the analytics team, so I think we should fix it if not so complex, than move forward to the right approach.

**Comment by LTruong2** [comment]
What is the update on this?  We currently don't have any data from world if shares in Adobe.
